src/main/resources/anniv5Resources/localization/eng/bladestormpack/Powerstrings.json

```diff
@@ -0,0 +1,37 @@
+{
+  "${ModID}:EscalatingBreezePower": {
+    "NAME": "Escalating Breeze",
+    "DESCRIPTIONS": [
+      "Retain up to #b",
+      " card this turn. Chosen Attacks will cost #b",
+      " cards this turn. Chosen Attacks will cost #b",
+      " less next turn."
+    ]
+  },
+  "${ModID}:DownwindBlowPower": {
+    "NAME": "Downwind Blow",
+    "DESCRIPTIONS": [
+      "Take #b50% less damage this turn.",
+      "Take #b50% less damage for #b",
+      " turns."
+    ]
+  },
+  "${ModID}:GaleForcePower": {
+    "NAME": "Gale-Force",
+    "DESCRIPTIONS": [
+      "Whenever you play an Attack, gain #yWindrush equal to its cost + #b",
+      ", and draw #b",
+      " card if it costs #b",
+      " cards if it costs #b",
+      " or more. NL NL ",
+      "  Windrush : Whenever you would receive unblocked Attack damage this turn, reduce it that much."
+    ]
+  },
+  "${ModID}:WindrushPower": {
+    "NAME": "Windrush",
+    "DESCRIPTIONS": [
+      "Whenever you would receive unblocked Attack damage this turn, reduce it by #b",
```

Comment:
Going by base game powers like Static Discharge I believe this should be `attack` (lowercase) rather than `Attack`. Conceptual difference being that `Attack` is usually reserved for referring to player Attack cards, whereas enemies `attack` the player.